tests/tables/colr.rs

```diff
@@ -56,49 +56,474 @@ fn basic() {
+        face.paint_color_glyph(GlyphId(93), 0, &mut vec_painter);
+        assert_eq!(vec_painter.0, vec![
+            PushClipBox(ClipBox { x_min: 0.0, y_min: 0.0, x_max: 1000.0, y_max: 1000.0 }),
+            OutlineGlyph(GlyphId(2)),
+            PushClip,
+            Paint(RadialGradient(166.0, 768.0, 0.0, 256.0, 166.0, 768.0, Pad, vec![
+                CustomStop(0.0, RgbaColor { red: 0, green: 128, blue: 0, alpha: 255 }),
+                CustomStop(0.5, RgbaColor { red: 255, green: 255, blue: 255, alpha: 255 }),
+                CustomStop(1.0, RgbaColor { red: 255, green: 0, blue: 0, alpha: 255 })])),
+            PopClip,
+            PopClip]
+        )
+    }
+
+    #[test]
+    fn rotate() {
+        let mut face = Face::parse(COLR1_VARIABLE, 0).unwrap();
+        face.set_variation(Tag::from_bytes(b"ROTA"), 150.0);
+        let mut vec_painter = VecPainter(vec![]);
+        face.paint_color_glyph(GlyphId(99), 0, &mut vec_painter);
+        assert!(vec_painter.0.contains(&Rotate(0.87341005)))
+    }
+
+    #[test]
+    fn rotate_around_center() {
+        let mut face = Face::parse(COLR1_VARIABLE, 0).unwrap();
+        face.set_variation(Tag::from_bytes(b"ROTA"), 150.0);
+        let mut vec_painter = VecPainter(vec![]);
+        face.paint_color_glyph(GlyphId(101), 0, &mut vec_painter);
+        assert!(vec_painter.0.contains(&Rotate(0.9336252)))
+    }
+
+    #[test]
+    fn skew() {
+        let mut face = Face::parse(COLR1_VARIABLE, 0).unwrap();
+        face.set_variation(Tag::from_bytes(b"SKXA"), 46.0);
+        let mut vec_painter = VecPainter(vec![]);
+        face.paint_color_glyph(GlyphId(103), 0, &mut vec_painter);
+        assert!(vec_painter.0.contains(&Skew(0.3944702, 0.0)));
+    }
+
+    #[test]
+    fn skew_around_center() {
+        let mut face = Face::parse(COLR1_VARIABLE, 0).unwrap();
+        face.set_variation(Tag::from_bytes(b"SKXA"), 46.0);
+        let mut vec_painter = VecPainter(vec![]);
+        face.paint_color_glyph(GlyphId(104), 0, &mut vec_painter);
+        assert!(vec_painter.0.contains(&Skew(0.3944702, 0.0)));
+    }
+
+    #[test]
+    fn transform() {
+        let mut face = Face::parse(COLR1_VARIABLE, 0).unwrap();
+        face.set_variation(Tag::from_bytes(b"TRDX"), 150.0);
+        let mut vec_painter = VecPainter(vec![]);
+        face.paint_color_glyph(GlyphId(109), 0, &mut vec_painter);
+
+        assert!(vec_painter.0.contains(&Transform(ttf_parser::Transform {
+            a: 1.0,
+            b: 0.0,
+            c: 0.0,
+            d: 1.0,
+            e: 274.9939,
+            f: 125.0
+        }
+        )));
+    }
+
+    #[test]
+    fn translate() {
+        let mut face = Face::parse(COLR1_VARIABLE, 0).unwrap();
+        face.set_variation(Tag::from_bytes(b"TLDX"), 100.0);
+        let mut vec_painter = VecPainter(vec![]);
+        face.paint_color_glyph(GlyphId(114), 0, &mut vec_painter);
+
+        assert!(vec_painter.0.contains(&Translate(99.975586, 100.0)));
+    }
+}
```

Comment:
New line.

Reply:
There should be a newline now.. I'm not sure why it doesn't show up.

Reply:
<img width="426" alt="image" src="https://github.com/RazrFalcon/ttf-parser/assets/47084093/edaa81cd-b22f-40a4-8677-73c2e3d18cb7">
At least in my editor it shows up now.